### PR TITLE
Collapse contributing and issues for clarity.

### PR DESCRIPTION
**Pull request by sciencewhiz**
Clarify that this is Control System documentation, as opposed to
all the other documentation at
https://www.firstinspires.org/resource-library/frc/competition-manual-qa-system

### CODE REVIEW

**Comment by Daltz333** [comment]
Note that the conf.py may need to be updated as well. I personally think this is lengthy, and should be obvious.

File: source/index.rst [hunk 1]
```diff
@@ -79,17 +79,13 @@ Welcome to the FIRST Robotics Competition Documentation! This documentation will
...
 .. toctree::
    :maxdepth: 1
-   :caption: Contributing
+   :caption: Contributing to Documentation
 
    docs/contributing/contribution-guidelines
    docs/contributing/style-guide
    docs/contributing/build-instructions
    docs/contributing/top-contributors
+   Report a documentation issue <https://github.com/wpilibsuite/frc-docs/issues>
```
Comment (Daltz333) [inline source/index.rst]: I don't like how less obvious this becomes.

Comment (Daltz333) [inline source/index.rst]: This is be kept in it's own toctree to emphasis issue distinction, and the URL can be changed to point to the allwpilib repo, where we can move issues around as needed.

**Comment by Kevin-OConnor** [comment]
> 
> 
> Note that the conf.py may need to be updated as well. I personally think this is lengthy, and should be obvious.

What would need to be updated in conf.py? I don't see anything

**Comment by Daltz333** [comment]
> 
> 
> > Note that the conf.py may need to be updated as well. I personally think this is lengthy, and should be obvious.
> 
> What would need to be updated in conf.py? I don't see anything

project_name may need to be updated, not that it automatically appends "documentation" to the end. This is also what I believe determines the browser meta name.

**Comment by Kevin-OConnor** [comment]
> project_name may need to be updated, not that it automatically appends "documentation" to the end. This is also what I believe determines the browser meta name.

It appears to be used to append to the name of the page in the ``<title>`` tag. So if we update the name of the index page as I have proposed in #249, the title that shows in a browser tab is "FIRST Robotics Competition Control System - FIRST Robotics Competition documentation." I don't have a problem changing it to something else, but nothing after "Competition" shows in Firefox or Chrome anyway

**Comment by Daltz333** [comment]
I actually kinda like that, it's not redundant.

**Pull request closed without merging**